sources/packages/api/src/services/workflow/workflow-actions.service.ts

```diff
@@ -159,6 +159,37 @@ export class WorkflowActionsService {
     }
   }
 
+  /**
+   * When an exception is detected in the student application dynamic data, for instance,
+   * when some document was uploaded and need to be reviewed, the workflow will stop till
+   * this message is received with the approval or denial from the Ministry user.
+   * @param applicationExceptionId exception id to send the message.
+   * @param status approval or denial status.
+   */
+  async sendApplicationExceptionApproval(
+    applicationExceptionId: number,
+    status: ApplicationExceptionStatus,
+  ): Promise<void> {
+    try {
+      await this.workflowService.sendMessage({
+        messageName: `sims-application-exception-approval-${applicationExceptionId}`,
+        all: false, // false means that the message is correlated to exactly one entity.
+        processVariables: {
+          exceptionsApprovalStatus: {
+            value: status,
+            type: "string",
+          },
+        },
+      });
+    } catch (error: unknown) {
+      this.logger.error(
+        `Error while sending application exception approval message applicationExceptionId: ${applicationExceptionId}.`,
+      );
+      this.logger.error(error);
+      // The error is not thrown here, as we are failing silently.
```

Comment:
Please move this comment to somewhere between line 184 and 185.

Reply:
Any specific reason? It is following exactly what the other 4 similar methods in the same file are doing.
The comment is placed exactly where the exception would be thrown. With a proper reason, we can change it.

Reply:
Yes, you are right, but maybe this is not the case. The comment is not about the implementation above it. The comment is there to explain why there is a try/catch that is capturing the exception and them doing nothing. Line 189 was supposed to have something like `throw error` that is missing in these methods. I am not sure it makes better sense now.